include/yaramod/types/literal.h

```diff
@@ -96,6 +103,12 @@ class Literal
 			os << literal._formatted_value.value();
 		else if (literal.isBool())
 			os << (literal.getBool() ? "true" : "false");
+		else if (literal.isSymbol())
+			os << literal.getSymbol()->getName();
+		else if (literal.isLiteralReference())
+		{
```

Comment:
Nitpick: there are no curly braces in other parts of this condition so I wouldn't expect them here either.

Reply:
I will fix it in my next PR, thank you for merging this.